www/js/app.js

```diff
@@ -2224,3 +2225,19 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     }
 
 });
+// navigation bar collapsing button logic 
+function collapBtn() {
+    var btn = document.getElementById("collapbtn");
+    var navIcons = document.getElementById("horizontalNavBtn");
+
+    btn.addEventListener('click', () => {
```

Comment:
As mentioned in CONTRIBUTING.md, we can't use arrow function syntax in this Repo yet, because we are still supporting old browsers and we are not yet using Babel to transpile. Please change.